packages/checkbox/src/checkbox.vue

```diff
@@ -176,16 +177,14 @@ export default defineComponent({
       }
     }
 
-    function handleChange(e: UIEvent) {
+    function handleChange(e: InputEvent) {
       if (isLimitExceeded.value) return
-      let value = ref(undefined)
-      if ((e.target as HTMLInputElement).checked) {
-        value.value = props.trueLabel === undefined ? true : props.trueLabel
-      } else {
-        value.value = props.falseLabel === undefined ? false : props.falseLabel
-      }
+      const target = e.target as HTMLInputElement
+      const value = target.checked
```

Comment:
Is there any reason that we allow `trueLabel` or `falseLabel` as `null`? Should we add a breaking change here? @JeremyWuuuuu @zouhangwithsweet @zazzaz